README_esri.md

```diff
@@ -0,0 +1,25 @@
+
+# How to build for runtimecore
+
+## linux
+
+```bash
+$ cd esri
+$ docker build --tag=opencv:4.7.0 - < opencv.dockerfile
+$ docker run --rm -it -u $(id -u):$(id -g) --volume ${HOME}/w/opencv:/opencv --workdir /opencv/esri opencv:4.7.0 bash
+$ bash build_opencv_linux.sh
```

Comment:
Since this is scripted, you don't even need to drop into the bash in the previous step. You can just call this script at the end of the docker line above instead of calling bash